src/App.tsx

```diff
@@ -1,61 +1,139 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
The `usersFromServer` data should not be directly imported here. Instead, it should be passed down as props from a parent component, such as `App`, to ensure better data flow and maintainability. This aligns with the best practices for managing data in React applications.